app/assets/javascripts/mirador_widget_admin.js

```diff
@@ -128,6 +128,10 @@
         showSourceLocationInput(block, sourceLocationValue(block));
       },
 
+      updateHiddenMiradorConfig(block) {
+        updateHiddenMiradorConfig(block);
+      },
```

Comment:
❓

Reply:
Yeah, just making the available globally.. could easily be refactored I guess.

Reply:
Oh, I see, I needed to see the diff above what GH was showing to see that this was in the returned object.  I would be 👌w/ just making the function public, but am totally fine with this too (revealing module pattern, I 👀 you).

Reply:
I've made the method public in b1c9cf4